src/container/container_iv.c

```diff
@@ -1092,6 +1084,40 @@ cont_iv_prop_fetch(struct ds_iv_ns *ns, uuid_t cont_hdl_uuid,
 	return rc;
 }
 
+/*
+ * exported APIs
+ */
+int
```

Comment:
Why not just rename the 'cont_iv_snapshots_fetch' to 'ds_cont_fetch_snaps'?

Reply:
yes, I just wanted to minimize the change, but I agree we don't have a lot of callers for these functions at all.